pkg/apis/planetscale/v2/vitesscell_types.go

```diff
@@ -189,6 +189,11 @@ type VitessCellGatewaySpec struct {
 	// Tolerations allow you to schedule pods onto nodes with matching taints.
 	// +kubebuilder:validation:EmbeddedResource
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+
+	// TopologySpreadConstraint can optionally be used to
+	// specify how to spread vtgate pods among the given topology
+	// *kubebuilder:validation:EmbeddedResource
```

Comment:
Does `*` vs `+` here have some special meaning I am not familiar with?

Reply:
from the docs - it should be `+`, so I fixed this one and the other stray `*`

https://book.kubebuilder.io/reference/markers/crd-validation.html